### DISCUSSION TITLE
do you have any plan to support websocket subscription / useSubscription hooks

### DISCUSSION THREAD

**Discussion by krunaldodiya** [blitz-js/blitz#2540] [Ideas]
it would be rather easy to have useSubscription hooks and user can directly configure the websocket provider, like socket.io or pusher and on frontend side just use const [subscribe, unsubscribe] = useSubscription('newUserCreated'), don't you think so ?

**Comment by flybayer** [comment]
Hey! Yes we do plan to add this at some point, but won't be a priority until after 1.0

There's been some previous discussions on this:
- https://github.com/blitz-js/blitz/discussions/1529
- https://github.com/blitz-js/blitz/discussions/2020
- https://github.com/blitz-js/blitz/discussions/1400
